packages/ra-language-french/src/index.ts

```diff
@@ -190,6 +190,7 @@ const frenchMessages: TranslationMessages = {
                 title: 'Inspecteur',
                 content: 'Sélectionner un composant pour le configurer',
                 reset: 'Réinitialiser',
+                unlabeled: 'Colonne sans libellé #%{column}',
```

Comment:
French translation is good :grin: 

But in this specific case I'd rather keep 'label' instead of 'libellé', to refer to the actual prop name rather than the notion

```suggestion
                unlabeled: 'Colonne sans label #%{column}',
```